pertpy/tools/_cinemaot.py

```diff
@@ -327,7 +327,7 @@ def get_dim(
         c: float = 0.5,
         use_rep: str = "X_pca",
     ):
-        """Estimating the rank of the count matrix. Always use adata.raw.X.
+        """Estimating the rank of the count matrix. Always use adata.raw.X. Make sure it is the raw count matrix.
```

Comment:
This can be checked with https://github.com/theislab/pertpy/blob/main/pertpy/tools/_distances/_distances.py#L677

We could consider checking for this and then throwing an error